### ISSUE TITLE
Can't open link if swapfile exists

### ISSUE THREAD

**Issue by Nudin** [vimwiki/vimwiki#466]
When I try to follow a link to a file for which a awapfile exists (because vim crashed/was killed earlier) vim freezes and after pressing enter again shows the error messages:
```
Fehler beim Ausführen von "function vimwiki#base#follow_link[58]..vimwiki#base#open_link[31]..vimwiki#base#edit_file":                                        
Zeile   21:
E325: ACHTUNG
Fehler beim Ausführen von "function vimwiki#base#follow_link[58]..vimwiki#base#open_link":
Zeile   31:
E171: Fehlendes :endif
Fehler beim Ausführen von "function vimwiki#base#follow_link":
Zeile   58:
E171: Fehlendes :endif
```
After another press of enter the file is opened but keybindings don't work.

**Comment by Holzhaus** [comment]
I had the same issue and had to delete the swapfile manually to fix the issue.

**Comment by Nudin** [comment]
Should be fixed in the dev-branch with 2185815b0
